### PR TITLE
Rework the function into the latest style and make sure it all works

### PR DESCRIPTION
**Pull request by 0xdevalias**
The function didn't seem to work as it was before. Updated to the latest style, make sure it works, follow faas-cli style.

### CODE REVIEW

File: youtube-dl/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+FROM wernight/youtube-dl
```
Comment (alexellis) [inline youtube-dl/Dockerfile:1]: Why did the base image change?

Comment (0xdevalias) [reply youtube-dl/Dockerfile:1]: The old one was 200mb and running a version from 2014, which was breaking on 'new' youtube videos. (see the 2nd commit message)

**Comment by 0xdevalias** [comment]
The dockerfile is simple enough if you wanted to fork and maintain it locally, though at some point that seems to negate half the purpose of shared docker images:
* https://hub.docker.com/r/wernight/youtube-dl/~/dockerfile/

There also seem to be a pile of other images.. maybe one of them would suit better:
* https://hub.docker.com/search/?isAutomated=1&isOfficial=0&page=1&pullCount=0&q=youtube-dl&starCount=0

**Comment by 0xdevalias** [comment]
Seems you copy/paste added most of the things I had here anyway..

**Pull request closed without merging**